src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -910,6 +916,11 @@ export class Workbench implements IPartService {
 		this.workbenchLayout.layout();
 	}
 
+	private setFontAliasing(enabled: boolean) {
+		this.fontAliasingEnabled = enabled;
+		this.workbench.style('-webkit-font-smoothing', enabled ? 'antialiased' : '');
```

Comment:
I think it would be better to apply this on the `<body>` element to make sure that really everything is covered. E.g. there are things that can draw outside of the workbench (the tweet a smile feedback dialog for example).